### DISCUSSION TITLE
Project Roadblock in Jupyter

### DISCUSSION THREAD

**Discussion by nguyen-peter** [Project-MONAI/MONAI#7138] [Q&A]
Hello,

I am (trying) using MONAI's unetr model to create an image segmentation model for CT scans. I have been following along [this guide](https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/unetr_btcv_segmentation_3d.ipynb) to help me in jupyterhub. Currently, the block of code I am trying to run does not produce an output, and causes my kernel to restart / crash. I don't really understand why. Here's the code for reference:

data_dir = "/panfs/jay/groups/25/barkerfk/nguy4214/"
split_json = "dataset_0.json"

datasets = data_dir + split_json
datalist = load_decathlon_datalist(datasets, True, "training", "/panfs/jay/groups/25/barkerfk/nguy4214/")
val_files = load_decathlon_datalist(datasets, True, "validation")
train_ds = CacheDataset(
    data=datalist,
    transform=train_transforms,
    cache_num=24,
    cache_rate=1.0,
    num_workers=8,
)
train_loader = DataLoader(train_ds, batch_size=1, shuffle=True, num_workers=8, pin_memory=True)
val_ds = CacheDataset(data=val_files, transform=val_transforms, cache_num=6, cache_rate=1.0, num_workers=4)
val_loader = DataLoader(val_ds, batch_size=1, shuffle=False, num_workers=4, pin_memory=True)

The dataset/JSON that I am loading isn't very large >200 lines, and the format of the JSON should be correct (I just replaced the example data with my own files). I'm not quite sure why the code isn't running.

**Comment by KumoLiu** [comment]
Hi @nguyen-peter, I don't see anything special in the code you posted. Could you please try to separate these codes into several cells and see which line crashes or just start with one data first?
Thanks!

**Reply by nguyen-peter** [reply]
Hi sorry for the late response - Based on some print statements, my code seems to be stopping because of the load_decathalon_datalist function. I don't quite understand why, as I've formatted my data the same way that the guide indicated.

**Reply by KumoLiu** [reply]
Hi @nguyen-peter, have you tried running the tutorial first, can you run it through?
